pkg/network/netlink/consumer.go

```diff
@@ -362,6 +363,10 @@ func (c *Consumer) initNetlinkSocket(samplingRate float64) error {
 		log.Debugf("rcv buffer size for netlink socket is %d bytes", size)
 	}
 
+	if err := c.socket.SetSockoptInt(unix.SOL_NETLINK, unix.NETLINK_LISTEN_ALL_NSID, 1); err != nil {
```

Comment:
Well, that explains it 😛 
Any thoughts about exposing this functionality behind a configuration flag?
The Conntrack code is arguably the main CPU hog on system-probe so I wonder if we this should be disabled by default.
I think most of of the Conntrack events we'll receive for different namespaces will be discarded as they won't contain translations (except for things like Istio)

Reply:
Yes, I have been thinking about doing that. Given that the istio use case is the only one at this point, I think a config flag is a good idea. Let me add that.